### DISCUSSION TITLE
`StructOptimizer` issue

### DISCUSSION THREAD

**Discussion by dohyeongkwon** [CederGroupHub/chgnet#121] [Q&A] [answered]
I can't import `CHGNet` and `StructOptimizer` from `chgnet.model`
I think that the GitHub code is expired so, it couldn't be available. 
I copied the source code of `ase.filters` from the `ase` website but it also can't solve this problem because of `ase.util.abc`
so how do I solve it?

**Comment by bowen-bd** [comment, marked as answer]
can you try `pip install git+https://gitlab.com/ase/ase`?

**Reply by dohyeongkwon** [reply]
Thank you for your kind response. The issue was due to a firewall problem with our cluster server. sorry for the inconvenience caused. If anyone else is experiencing a similar issue, i recommend using SSH or directly downloading the 3.23.0tar.gz file and uploading it to the server.
